era_5g_interface/h264_encoder.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from pathlib import Path
+
+import av
+import numpy as np
+from av.codec import CodecContext
+from av.video.codeccontext import VideoCodecContext
+
+Path("input").mkdir(parents=True, exist_ok=True)
+
+logger = logging.getLogger("H264 encoder")
+
+
+class H264Encoder:
+    def __init__(self, fps: float, width: int, height: int) -> None:
+        self.encoder: VideoCodecContext = CodecContext.create("h264", "w")
+        self.encoder.width = width
+        self.encoder.height = height
+        self.encoder.framerate = fps
+        self.encoder.pix_fmt = "yuv420p"
+        # self.encoder.options = {"crf": "0", "preset": "ultrafast", "tune": "zerolatency"}
```

Comment:
Remove, make it an option, or document why it is commented if it makes sense to keep it here.